scripts/check-nix-is-updated.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+# check and warn if `pkgs/default.nix` is out of date
```

Comment:
References to `pkgs/default.nix` and `./pkgs/generate.sh` don't make sense to have in stack2nix.